cmd/versionhook/main.go

```diff
@@ -20,7 +20,7 @@ import (
 
 const (
 	agentStatusFilePathEnv = "AGENT_STATUS_FILEPATH"
-	logFilePathEnv         = "PRE_STOP_HOOK_LOG_PATH"
+	logFilePathEnv         = "VERSION_UPGRADE_HOOK_LOG_PATH"
```

Comment:
Suggestion: instead of writing the log to a file we could simply write it to stdout and let Kubernetes pick up on the logs. This would make debugging easier as the log currently is unavailable when `mongod` is stopped and the log file will also be removed during a restart.

Reply:
yes this was valid for enterprise but i thought all logs were in stdout

Reply:
Cool. Changed now to write directly to stdout